shared/oae/bundles/email/default.properties

```diff
@@ -8,4 +11,8 @@ ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_MULTIPLE_ACTORS = New activity is waiting for y
 ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_SINGLE_ACTOR_MULTIPLE_ACTIVITIES = ${actorLink} has been very active recently
 ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_SINGLE_ACTOR_SINGLE_ACTIVITY = ${actorLink} has been active recently
 ACTIVITY_EMAIL_SUMMARY_WEEKLY = Here's what happened in the last week
-
+EMAIL_VERIFY_ADMIN_BODY_EXPLANATION = A request to change the email address of your account has been made by an administrator. If you’d like to use this email address instead, please click the following link.
```

Comment:
> We have received a request to change the email address of your account. Please click the button to verify your email address. If you did not request this change, please ignore this message.

There was already quite a bit of discussion about what this message should be. Recall there was some discussion around whether or not we should have a different message based on whether or not an administrator was the one who changed the email address. We settled on the messaging that we have right now so I'm hesitant to change it. I'll just update the layout for now.